.github/workflows/pull_request.yml

```
@@ -33,7 +33,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Tag
-        run: echo "TAG=${{ github.event.pull_request.head.ref }}" >> $GITHUB_ENV
+        run: |
+          CLEAN_TAG=$(echo "${{ github.event.pull_request.head.ref  }}"  | tr / -)
```

Bit of a nit but there is extra space in this line that we don't need
```suggestion
          CLEAN_TAG=$(echo "${{ github.event.pull_request.head.ref }}" | tr / -)
```